src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackStateMapperTest.java

```diff
@@ -0,0 +1,317 @@
+package org.fogbowcloud.manager.core.plugins.cloud.openstack;
+
+import org.fogbowcloud.manager.core.models.instances.InstanceState;
+import org.fogbowcloud.manager.core.models.instances.InstanceType;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class OpenStackStateMapperTest {
+
+    @Test
+    public void testComputeActiveStatusToReady() {
+        InstanceType instanceType = InstanceType.COMPUTE;
+
+        InstanceState instanceState1 = OpenStackStateMapper.map(instanceType, "active");
+        InstanceState instanceState2 = OpenStackStateMapper.map(instanceType, "ACTIVE");
+
+        Assert.assertEquals(instanceState1, InstanceState.READY);
+        Assert.assertEquals(instanceState2, InstanceState.READY);
+    }
+
```

Comment:
@llvieira should we protect the code agains a null parameter? the code is throwing a null point exception in such case.

Reply:
I don't think this defensive programming is needed here, since we always pass a predefined `instanceType` and the `openStackState` will came from the cloud.